onyo/lib/onyo.py

```diff
@@ -337,19 +358,29 @@ def _get_files_untracked(self) -> set[Path]:
         untracked = {Path(x) for x in self._git(['ls-files', '-z', '--others', '--exclude-standard']).split('\0') if x}
         return untracked
 
-    def _get_root(self) -> Path:
+    @staticmethod
+    def _find_root(directory: Union[Path, str]) -> Path:
         """
+        Find and return the root of an Onyo repository (containing `.git/` and
+        `.onyo/`) for any path inside of an existing repository.
+
+        If the `directory` is not inside an existing repository, an error is
+        raised.
         """
+        directory = Path(directory).resolve()
+        root = Path()
```

Comment:
Why initialize `root` with `'.'`?
In case you're thinking that `Path()` is some sort of empty path - it's not:

```
In [1]: from pathlib import Path

In [2]: Path()
Out[2]: PosixPath('.')
```

So, if code below works as expected, the initial value is never relevant at all. If it isn't, we end up treating CWD as root, because it happens to have a `.onyo` subdir. Point being: This can only go wrong but doesn't provide a benefit as far as I can tell. Or is there a case where CWD is intended to be some sort of fallback?